package.json

```diff
@@ -36,17 +36,11 @@
         "docs:review": "alex packages/**/*.md",
         "docs:start": "yarn workspace documentation serve --watch",
         "find": "test -f custom-elements.json",
-        "format:css": "yarn lint:css --fix && pretty-quick --pattern \"{packages,tools}/**/*.css\"",
         "gen-react-wrapper": "node ./tasks/gen-react-wrapper.js",
         "icons": "wireit",
         "icons:ui": "wireit",
         "icons:workflow": "wireit",
-        "lint": "run-p lint:js lint:docs lint:ts lint:css lint:packagejson",
-        "lint:css": "stylelint \"packages/**/*.css\" \"tools/**/*.css\"",
-        "lint:docs": "eslint -f pretty \"projects/documentation/**/*.ts\"",
-        "lint:js": "pretty-quick --pattern \"tasks/**/*.js\" && pretty-quick --pattern \"scripts/**/*.js\"",
-        "lint:packagejson": "pretty-quick --pattern package.json --pattern \"packages/*/package.json\" --pattern \"projects/*/package.json\" --pattern \"tools/*/package.json\" --pattern \"react/*/package.json\"",
-        "lint:ts": "pretty-quick --pattern \"packages/**/*.ts\" && eslint -f pretty \"packages/**/*.ts\" && pretty-quick --pattern \"tools/**/*.ts\" && eslint -f pretty \"tools/**/*.ts\"",
+        "lint": "git status --porcelain && git add . && yarn lint-staged",
```

Comment:
This git status command checks first if there are uncommitted assets and adds them before linting.